services/web/client/source/class/osparc/theme/Appearance.js

```diff
@@ -380,6 +379,23 @@ qx.Theme.define("osparc.theme.Appearance", {
       })
     },
 
+    "strong-button": {
+      include: "material-button",
+      style: state => ({
+        decorator: state.hovered ? "strong-bordered-button" : "no-border",
+        backgroundColor: "strong-main"
+      })
+    },
+
+    "danger-button": {
+      include: "material-button",
+      style: state => ({
+        decorator: "bordered-button",
+        backgroundColor: state.hovered ? "danger-red" : null,
```

Comment:
I guess this might change when Bruno comes out with the "delete theme"?

Reply:
Correct, he will soon provide as with the proper ```danger-red```